ovirt_imageio/client/_options.py

```diff
@@ -24,6 +24,9 @@
 import uuid
 
 
+DISK_TIMEOUT = 120
```

Comment:
If the timeout is only used for waiting until the engine has created the new disk then maybe we could give it a more descriptive name, like DISK_CREATION_TIMEOUT or something like that, because in the general context of "DISK" there could be different timeouts of other operations as well.